Sources/SwiftCompilerPluginMessageHandling/CompilerPluginMessageHandler.swift

```diff
@@ -212,3 +241,31 @@ extension PluginProvider {
     throw UnimplementedError()
   }
 }
+
+#if compiler(>=6) && os(WASI)
+
+/// A callback invoked by the Wasm Host when new data is available on `stdin`.
+///
+/// This is safe to access without serialization as Wasm plugins are single-threaded.
+nonisolated(unsafe) private var readabilityHandler: () -> Void = {
+  fatalError(
+    """
+    CompilerPlugin.main wasn't called. Did you annotate your plugin with '@main'?
+    """
+  )
+}
+
+@_expose(wasm,"swift_wasm_macro_v1_pump")
```

Comment:
Incorrect formatting on the attribute arguments will be fixed by https://github.com/apple/swift-format/pull/750